clippy_utils/src/ty.rs

```diff
@@ -114,7 +115,7 @@ pub fn has_iter_method(cx: &LateContext<'_>, probably_ref_ty: Ty<'_>) -> Option<
 
 /// Checks whether a type implements a trait.
 /// The function returns false in case the type contains an inference variable.
-/// See also `get_trait_def_id`.
+/// See also [`get_trait_def_id`].
```

Comment:
This link only works for me with the direct path, as it's not in the current scope. This should fix it:

```suggestion
///
/// See also [`get_trait_def_id`](super::get_trait_def_id).
```

These small doc updates are also very very and can help contributors in the future. Thank you for making them :)

Reply:
darn it! I thought I checked the rustdoc output to check the link was right. Clearly someone (perhaps me) needs to go fix all the other warnings that buried it haha.